Lab_day-3_Dorota_Parina.js

```diff
@@ -0,0 +1,83 @@
+// Iteration 1: Names and Input
+
+var hacker1 = "Parina";
+console.log(`The driver's name is ` + hacker1);
+
+var hacker2 = "Dorota";
+console.log(`The navigator's name is ` + hacker2);
+
+
+// Iteration 2: Conditionals
+
+function theLongestName(name) {
+    if (hacker1.length > hacker2.length) {
+        return "The Driver has the longest name, it has " + hacker1.length + " characters"
+    } else if (hacker1.length < hacker2.length) {
+        return "Yo, navigator got the longest name, it has " + hacker2.length + " characters"
```

Comment:
Here you are only comparing the first letter and not the whole word, use a loop instead to parse each letter of the word!